### PR TITLE
Add section about Hazelcast Cloud CLI

### PR DESCRIPTION
**Pull request by nosan**

### CODE REVIEW

File: docs/modules/ROOT/pages/cli.adoc [hunk 1]
```diff
@@ -0,0 +1,99 @@
+= Hazelcast Cloud CLI
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/cli.adoc:1]: Should this now be the Hazelcast Viridian CLI?

File: docs/modules/ROOT/pages/cli.adoc [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+Hazelcast Cloud CLI is known as `hzcloud` is a command line tool to make actions on Hazelcast Cloud easily. Hazelcast Cloud offers the leading in-memory computing platform, Hazelcast IMDG, as a fully managed service that integrates with your existing virtual private cloud.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/cli.adoc:3]: I don't think we need to describe Hazelcast Viridian here. This page is about the CLI. We also shouldn't be writing about IMDG here.

File: docs/modules/ROOT/pages/cli.adoc [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+== Installing `hzcloud`
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/cli.adoc:5]:
```suggestion
== Installing the Hazelcast Viridian CLI
```

I don't think we should use `hzcloud` as the product name. Users will not search Google for this.

File: docs/modules/ROOT/pages/cli.adoc [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+wget https://github.com/hazelcast/hazelcast-cloud-cli/releases/latest/download/hzcloud-linux-amd64 -O /usr/local/bin/hzcloud && chmod +x /usr/local/bin/hzcloud
+----
+**Windows**
+
+[source,sh]
+----
+curl -o hzcloud.exe `
+  https://github.com/hazelcast/hazelcast-cloud-cli/releases/latest/download/hzcloud-windows-amd64
+----
+On Windows, in order to use `hzcloud` on everywhere you need to put `hzcloud.exe` into your PATH.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/cli.adoc:30]: Is there a command we can share that users can copy/paste?

File: docs/modules/ROOT/pages/cli.adoc [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+----
+
+==  Examples
+You can use `hzcloud` to interact with resources on **Hazelast Cloud**. You can find some examples to begin with.
+
+- **Create a Cluster**
+
+[source,sh]
+----
+hzcloud serverless-cluster create --name testcluster  --region us-west-2
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/cli.adoc:61]: Is there a way to specify development/production clusters?

File: docs/modules/ROOT/pages/cli.adoc [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+Aliases:
+  serverless-cluster, slc
+
+Available Commands:
+  create         This command allows you to create a serverless Hazelcast cluster.
+  custom-classes This command allows you to manage custom classes on your serverless cluster like: list, upload, delete.
+  delete         This command allows you to delete a serverless Hazelcast cluster.
+  get            This command get detailed configuration of a serverless Hazelcast cluster.
+  list           This command allows you to create a serverless Hazelcast cluster.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/cli.adoc:87]: Starting each line with 'This command' is distracting. We should be starting with the verb.

Also the description for `list` is wrong.

File: docs/modules/ROOT/pages/cli.adoc [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+This command allows you to make actions on your serverless clusters like: create, delete, get, list, stop or resume.
+
+Usage:
+  hzcloud serverless-cluster [command]
+
+Aliases:
+  serverless-cluster, slc
+
+Available Commands:
+  create         This command allows you to create a serverless Hazelcast cluster.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/cli.adoc:83]: According to our brand guidelines, we should refer to clusters as Viridian Serverless clusters, not serverless Hazelcast clusters.

**Review by JakeSCahill — CHANGES_REQUESTED** [review]
General comment: Why are we excluding the other commands? Users will find them anyway when they download the CLI. Won't it be frustrating that we don't document the other commands?

In the staging docs for the Serverless beta launch, we provide a link to the CLI: https://github.com/hazelcast/cloud-docs/blob/staging/docs/modules/ROOT/pages/tools.adoc

Would this be enough for now?

**Comment by nosan** [comment]
Thanks @JakeSCahill 
The problem is that we don't have CLI for viridian at all :( CLI works only for serverless and enterprise clusters.
Maybe we need to put this on hold and just leave a link to the GitHub page.

**Pull request closed without merging**